api/create_file_event.go

```diff
@@ -16,12 +16,12 @@ func HandlerCreateFileEvent(createFileEvent CreateFileEvent) http.HandlerFunc {
 		var event sdk.FileEvent
 
 		if err := json.NewDecoder(req.Body).Decode(&event); err != nil {
-			writeError(w, buildErrors(err, "BadJsonEncoding"), http.StatusBadRequest)
+			writeError(w, buildGenericError("BadJson", "The JSON is not in a valid format"), http.StatusBadRequest)
```

Comment:
This error should also be logged using log.error, which should also record the request details so it is clear should we see any support requests regarding this error.